lib/elixir/lib/module.ex

```diff
@@ -830,6 +830,10 @@ defmodule Module do
         false ->
           {name, arity} = tuple
           raise "cannot make function #{name}/#{arity} overridable because it was not defined"
```

Comment:
I would have pushed that to master later on, but I can do it in this PR. Up
to you :)

On Thursday, 23 June 2016, José Valim <notifications@github.com> wrote:

> In lib/elixir/lib/module.ex
> <https://github.com/elixir-lang/elixir/pull/4856#discussion_r68276104>:
>
> > @@ -830,6 +830,10 @@ defmodule Module do
> >          false ->
> >            {name, arity} = tuple
> >            raise "cannot make function #{name}/#{arity} overridable because it was not defined"
>
> no ArgumentError here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/4856/files/9da8e4ded93edc02dac47edd3666a2056cda7741#r68276104>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADtcShN70wVaSnDqKasZanijO2mbun5Kks5qOr_3gaJpZM4I8U6Y>
> .
>


-- 

Andrea Leopardi
an.leopardi@gmail.com